change/@ni-nimble-components-278f4b89-b1f8-402d-a3e7-ba0338b1b348.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The problem was really that the selected styling wasn't being applied to the active option. The arrow keys and typeahead were still changing the active option, there just wasn't a visual reflection of it. This required (afaik) adding styles to the `ListOptionGroup` to handle the styling of slotted `ListOption`s. This is a concern because they currently rely on the new `activeOption` attribute, which is not implemented by the `Combobox`. I could not figure out how to style the slotted `ListOption`s (within a `ListOptionGroup`) from the `Select` style, which would avoid that concern.

Reply:
This might run into the same issue of combobox & select needing different behavior, but could the list option know how to style itself when `activeOption` is set? Then the exact same behavior would be achieved regardless of whether or not the parent is a group or a select?

Reply:
I've at least tried this before, but with the `ListOption` already styling itself with respect to `aria-selected='true'` has proven tricky to mix that with `activeOption` as well.

Reply:
I am seeing this bug again in the latest build: the arrow keys don't work when there are option groups.
https://60e89457a987cf003efc0a5b-urkleipuso.chromatic.com/?path=/docs/components-select--docs

Reply:
Yep. I've corrected the issue and added a matrix test similar to the one I had made for options with no groups.